src/qt/macdockiconhandler.mm

```diff
@@ -126,3 +121,12 @@ void setupDockClickHandler() {
 
     emit this->dockIconClicked();
 }
+/**
+ * Force application activation on macOS. With Qt 5.5.1 this is required when
+ * an action in the Dock menu is triggered.
+ * TODO: Define a Qt version where it's no-longer necessary.
+ */
+void ForceActivation()
```

Comment:
Looks like you are missing something, this method is not called anywhere

Reply:
Ah yes,my bad,will remove that since it isnt used.

Reply:
First check in btc sources where it's used, the "why" it's pretty clear in the comments above the method.